tests/integration/README.md

```diff
@@ -23,7 +23,7 @@ Some tests carry requirements. We're testing the real service here, not just usi
 The test suite expects these environment variables to be present:
 
 ```shell
-    export SMARTSHEET_ACCESS_TOKEN="SSSSSSSSSSSSSSSSSSSSSSSSSS"
+    export SMARTSHEET_ACCESS_TOKEN="Bearer 3JRm067VUxw1u84MRMHEguCYdn5m8nvFpVhWb"
```

Comment:
For security reasons we should never push tokens to version control. Let's remove this Salman.